app/controllers/my/profiles_controller.rb

```diff
@@ -0,0 +1,31 @@
+class My::ProfilesController < My::BaseController
+ before_action :authenticate_volunteer!
+
+  def index
```

Comment:
we can remove (on another PR) the other actions, and leave `edit` and `update`. Because we are not going to use the others.